docs/usage/configuration-options.md

```diff
@@ -367,13 +367,61 @@ The Dependency Dashboard therefore provides visibility as well as additional con
 
 ## dependencyDashboardApproval
 
-Setting `dependencyDashboardApproval` to `true` means that Renovate will no longer create branches/PRs automatically but instead wait for manual approval from within the Dependency Dashboard.
+By default Renovate creates branches/PRs automatically whenever a package gets an update.
 
-In this case, the Dependency Dashboard _does_ change the flow of Renovate, because PRs will stop appearing until you approve them within the issue.
-Instead of enabling this repository-wide, you may instead with to use package rules to enable it selectively, e.g. for major updates only, or for certain package managers, etc.
-i.e. it is possible to require approval for only certain types of updates only.
+You can override the default behavior by setting `dependencyDashboardApproval` to `true`.
+Renovate waits for your approval from the Dependency Dashboard before creating a PR.
+You can approve a pending PR by ticking the checkbox in the Dependency Dashboard issue.
 
-Note: Enabling Dependency Dashboard Approval implicitly enables `dependencyDashboard` too, so it is not necessary to configure both to `true`.
+Note: When you set `dependencyDashboardApproval` to `true` the Dependency Dashboard issue will be created automatically, you do not need to turn on the `dependencyDashboardApproval` explictly.
```

Comment:
Maybe we should tell readers to turn on Dependency Dashboard as well in the config when they are using `dependencyDashboardApproval`? This will probably prevent a bunch of edge-cases?

If we want to tell them to turn on the Dashboard explicitly, I'll also change the JSON examples below to include turning on the dashboard explicitly.

Reply:
This sentence doesn't seem to make sense right now. Did you mean they didn't need to turn on dependencyDashboard explicitly ?

Reply:
Whoops, I was getting my `dependencyDashboardApproval` and the plain `dependencyDashboard` mixed up.
Pushed a commit to fix the nonsensical sentence. 😄